pkg/utils/ebpf/ebpf.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 The Inspektor Gadget authors
```

Comment:
```suggestion
// Copyright 2025 The Inspektor Gadget authors
```

Reply:
I'll skip it to avoid a whole CI run again.